src/svg/axis.js

```diff
@@ -123,6 +123,15 @@ export default function() {
 
             g.select('text')
                .attr('transform', translate(0, labelOffset))
+               .attr('dy', function() {
+                   var offset = '0em';
```

Comment:
The same values are also used by D3's axis ([`.71em`](https://github.com/mbostock/d3/blob/c36befc7361cd61538294048c387a8a407b65dd1/src/svg/axis.js#L56) and [`.32em`](https://github.com/mbostock/d3/blob/c36befc7361cd61538294048c387a8a407b65dd1/src/svg/axis.js#L60)). As D3 uses them, I thought it was a fairly safe option!


Also, through trial and error, I tried various combinations of `dominant-baseline` and `dy` values, and these seem to work.

For example, in Chrome, with the axis orient set to bottom: `dominant-baseline: auto` and `dy="0.71em"` renders the same as `dominant-baseline: hanging` and `dy="0em"` (`dy="0.68em"` was slightly too close to the axis).